workflow-templates/im-test-postman.yml

```diff
@@ -0,0 +1,59 @@
+name: Run Postman Tests
+
+# TODO:  In order to use this workflow a few changes need to be made to the postman test configuration:
+# 1. Make the following npm package changes
+#    npm uninstall newman-reporter-teamcity
+
+# 2. The reporters and options need to be updated. For instance:
+#    old: "newman run tests/postman-tests/ParticipantGuidance.postman_collection.json -e tests/postman-tests/participant-guidance-prod.postman_environment.json -r cli,teamcity",
+#    old: "newman run tests/postman-tests/ParticipantGuidance.postman_collection.json -e tests/postman-tests/participant-guidance-prod.postman_environment.json -r cli,json --reporter-json-export postman-results.json",
```

Comment:
Should the second `old` be `new`? It kinda looks like we're going for a new/old comparison here, but I could be misunderstanding.

Reply:
You are correct, I'll fix that.